src/lib/pipeline.py

```diff
@@ -327,20 +355,26 @@ def output_table(self, data: DataFrame) -> DataFrame:
 
     @staticmethod
     def _run_wrapper(
-        source_cache_aux: Tuple[DataSource, Dict[str, str], Dict[str, DataFrame]]
+        output_folder: Path,
+        cache: Dict[str, str],
+        aux: Dict[str, DataFrame],
+        data_source: DataSource,
     ) -> Optional[DataFrame]:
         """ Workaround necessary for multiprocess pool, which does not accept lambda functions """
-        data_source, cache, aux = source_cache_aux
         try:
-            return data_source.run(cache, aux)
-        except Exception as exc:
-            warnings.warn("Error running data source {}".format(data_source.__class__.__name__))
+            return data_source.run(output_folder, cache, aux)
+        except Exception:
+            data_source_class = data_source.__class__.__name__
```

Comment:
I'm probably being nitpicky here, but this is technically a class name not a class so IMHO "data_source_class_name" might be a better name.

Reply:
Let's settle on `data_source_name` :-)